tools/tf2ngraph.py

```diff
@@ -197,7 +193,6 @@ def convert(inp_format, inp_loc, out_format, out_loc, outnodes):
    """
     assert inp_format in allowed_formats['input']
     assert out_format in allowed_formats['output']
-    assert ngraph_bridge.is_grappler_enabled()
```

Comment:
TODO: this assert should probably stay. Also add an assert in the pytest that the converted model has an ngraphencapsulate op in it